pylti/tests/test_flask_app.py

```diff
@@ -164,3 +164,12 @@ def post_grade2(grade, lti):
     """
     ret = lti.post_grade2(grade)
     return "grade={}".format(ret)
+
+
+@app.route("/default_lti")
+@lti(app=app)
```

Comment:
this should change to just @lti either here once #31 lands, or in #31 once this lands.